app/parsers/bulkrax/parser_export_record_set.rb

```diff
@@ -173,7 +173,7 @@ def collections
       # @see https://github.com/samvera/hyrax/blob/64c0bbf0dc0d3e1b49f040b50ea70d177cc9d8f6/app/indexers/hyrax/work_indexer.rb#L15-L18
       def file_sets
         @file_sets ||= ParserExportRecordSet.in_batches(candidate_file_set_ids) do |batch_of_ids|
-          fsq = "has_model_ssim:#{Bulkrax.file_model_internal_resource} AND id:(\"" + batch_of_ids.join('" OR "') + "\")"
+          fsq = "has_model_ssim:\"#{Bulkrax.file_model_internal_resource.demodulize}\" AND id:(\"" + batch_of_ids.join('" OR "') + "\")"
```

Comment:
What does `#demodulize` do?

Reply:
It removes the module from a class name. So Hyrax::FileSet would become just FileSet. Which is how Solr stores filesets. If there isn't a module to remove it will do nothing.